include/rocksdb/env.h

```diff
@@ -594,6 +614,18 @@ class RandomAccessFile {
     return Status::OK();
   }
 
+  // Read a bunch of blocks as described by reqs. The blocks can
+  // optionally be read in parallel. This is a synchronous call, i.e it
+  // should return after all reads have completed. The reads will be
+  // non-overlapping
+  virtual void MultiRead(ReadRequest* reqs, size_t num_reqs) {
```

Comment:
It does seem odd to me to not return Status for this API (outside of `ReadRequest`). Almost all the other main Env APIs return Status. 
Also, if we want this to generic enough, shouldn't we take into consideration the environments which inherently support multiread and return a single success/failure?

Reply:
The Status inside ReadRequest determines if that specific read was successful, but I believe there should be a way to determine the state of the entire API call itself.

Reply:
I see that io_submit() can return a status for the entire call, in addition to a status for each request. So I guess it might make sense to also have a return status for this API (in addition to the one in ```ReadRequest```), if nothing else than for logging purposes.

Reply:
Thanks for making the suggested change. The interface looks good to me.